addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2972,7 +2967,15 @@ var SnippetsMenu = Widget.extend({
      * @private
      * @param {jQuery}
      */
-    _patchForComputeSnippetTemplates($html) {},
+    _patchForComputeSnippetTemplates($html) {
+        // TODO: remove in master. Updates icons and images in non-editable
+        // environments to be editable thanks to the o_editable_media class
+        const addVXML = function(index, elem) { 
+            elem.dataset.vxml = "001";
+        };
+        $html.find('div.s_social_media').each(addVXML).find('i').addClass('o_editable_media');
+        $html.find('section.s_company_team').each(addVXML).find('img').addClass('o_editable_media');
```

Comment:
We try not to write new code using jQuery. I suggest this, tell me if it works:
```js
for (const snippet of $html[0].querySelectorAll('div.s_social_media, section.s_company_team')) {
    snippet.dataset.vxml = "001";
    for (const media of snippet.querySelectorAll('i, img')) {
        media.addClass('o_editable_media');
    }
}
```

Reply:
It doesn't. But it's only missing an iteration over each object in `$html` and filtering out nodes that aren't elements. Should be good now. 
So, no new jQuery at all then ? Unless fixing existing jQuery lines perhaps ?